modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -40,22 +34,23 @@ Here is an example of a Mule configuration that utilizes this to expose an HTTPS
 </http:listener-config>
 ----
 
-You may also refer to this knowledge-base article: https://support.mulesoft.com/s/article/Sample-App-Configuring-HTTPS-endpoint-for-deployment-in-Cloudhub[Sample App: Configuring HTTPS endpoint for deployment in CloudHub]
+You can also refer to this knowledge-base article: https://support.mulesoft.com/s/article/Sample-App-Configuring-HTTPS-endpoint-for-deployment-in-Cloudhub[Sample App: Configuring HTTPS endpoint for deployment in CloudHub]
```

Comment:
"can" has some of the same problems as may (permission? capability?) Why not just provide the link in a See Also at the end?